.github/workflows/build_docker.yaml

```diff
@@ -0,0 +1,40 @@
+name: Docker
+
+on:
+  pull_request:
+    branches:
+      - master
+      - production
+
+jobs:
+  pre_job:
+    name: Pre Job
+    runs-on: ubuntu-latest
+    outputs:
+      has_tfcli_change: ${{ steps.skip_check.outputs.tfcli }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: skip_check
+        with:
+          list-files: csv
+          filters: |
+            tfcli:
+              - 'operations/docker-compose.yml'
+              - 'operations/Dockerfile'
+              - '.github/workflows/build_docker.yml'
+
+  build_tfcli:
```

Comment:
This job may be redundant, because technically Docker Compose will build the container if doesn't exist in the registry, but I'd rather have it explicitly defined as an Action for clarity.

Reply:
agreed